packages/webawesome/src/components/card/card.css

```diff
@@ -102,3 +102,27 @@
 :host(:not([with-media])) .media {
   display: none;
 }
+
+/* Orientation Styles */
+:host([orientation='horizontal']) {
+  flex-direction: row;
+
+  .media {
+    border-start-start-radius: var(--inner-border-radius);
+    border-end-start-radius: var(--inner-border-radius);
+    border-start-end-radius: 0;
+  }
+}
+
+:host([orientation='horizontal']) ::slotted(*) {
+  display: block;
+  height: 100%;
+  margin: 0;
+  object-fit: cover;
+}
```

Comment:
```suggestion
:host([orientation='horizontal']) ::slotted(*) {
  display: block;
  height: 100%;
  margin: 0;
  object-fit: cover;
}
```

I'm not sure we want these styles to target all slotted elements. That results in issues like this when there are multiple elements in the default slot:
<img width="711" height="352" alt="Screenshot 2025-09-04 at 4 57 05 PM" src="https://github.com/user-attachments/assets/5f1886c4-f954-4699-a10f-2e339729b029" />

Seems like `display`, `height`, and `margin` should apply to `.body` and `object-fit` should apply to only the elements slotted into `.media`, right?

Reply:
Yeah, the plan is to have both header and footer actions. Problem is, you can't nest slots, so it would structurally change the card. That seems like a major change, but if we can explain it, it should be fine.

Reply:
Oooh, thanks for catching that. that's been updated